### DISCUSSION TITLE
How to add a new static website to Azuracast box (virtual host)?

### DISCUSSION THREAD

**Discussion by ncfernan** [AzuraCast/AzuraCast#4237] [Q&A]
I have a static website hosted with GoDaddy that I'd like to move to my Azuracast box. I'm trying to follow the help page for multi-site installation, but it seems that my `docker-composer.yml` file is missing something.

What I need is to serve the URL `ncfradio.com.br` using ports 80 and 443 as a virtual server in Nginx.

The wordpress example shown in Azuracast help pages assumes that there is a wordpress image to be retrieved and installed, but in my case I have just a bunch of static files that have to be served and they are already available.

My first attempt was:

file: `/var/ncfradio.com.br/.env`

```
COMPOSE_PROJECT_NAME=ncfradio_com_br

NCFRADIO_COM_BR_HTTP_PORT=80
NCFRADIO_COM_BR_HTTPS_PORT=443

NGINX_TIMEOUT=1800
NCFRADIO_COM_BR_VERSION=stable
#LETSENCRYPT_HOST=ncfradio.com.br
#LETSENCRYPT_EMAIL=ncfradio@gmail.com
```

file: `/var/ncfradio.com.br/docker-compose.yml`

```
services:
    ncfradio_com_br:
        container_name : ncfradio_com_br
        image : "ncfradio_com_br:latest"
        ports :
          - 80, 443
        environment:
            LANG : pt_BR.UTF-8
            NGINX_TIMEOUT : 1800
            VIRTUAL_HOST: ncfradio.com.br
            # LETSENCRYPT_HOST: ncfradio.com.br
            # LETSENCRYPT_EMAIL: ncfradio@gmail.com
        volumes:
          - /NCF_Radio/sites:/var/ncfradio.com.br/www
        networks:
         - azuracast_frontend
         - backend
        restart: always

networks:
    azuracast_frontend:
        external: true
    backend:
        driver: bridge

#volumes:
#    - /NCF_Radio/sites:/var/ncfradio.com.br/www

volumes :
    web_data : { }
```

Of course when I try to spin up the new service with `docker-compose up -d` an error message is shown:

```
Pulling ncfradio_com_br (ncfradio_com_br:latest)...
ERROR: The image for the service you're trying to recreate has been removed.
```

I know that I don't have an image, but if I comment the image line I get another type of error:

```
ERROR: The Compose file is invalid because:
Service ncfradio_com_br has neither an image nor a build context specified. At least one must be provided.
```

So, now I'm stuck. I don't have the faintest idea on how to add a static website to my Azuracast box. This should be much simpler than spin up a wordpress website...

Is there an example that I could use as reference?

**Comment by ncfernan** [comment]
Well, I did it. It was easier than I thought...

**Reply by BusterNeece** [reply]
@ncfernan Sorry for the delay; I had meant to help out with this when I had a chance but then I was sidetracked with medical issues. Glad you were able to get it figured out.

**Reply by ncfernan** [reply]
@SlvrEagle23 Please don't mind, it was fun to learn from scattered info on the net. Now I know a bit more about containers and docker.

Keep up your amazing work!

Edit: I used the word "bother" when I meant "mind"... sorry about that...
